addons/website_blog/static/src/js/options.js

```diff
@@ -45,6 +41,10 @@ options.registry.CoverProperties.include({
      */
     updateUI: async function () {
         await this._super(...arguments);
+        const isBlogCover = this.$target[0].className.includes('o_wblog_post_page_cover');
```

Comment:
`classList.contains` ? :)

Reply:
@qsm-odoo I wanted to match both `o_wblog_post_page_cover` and `o_wblog_post_page_cover_regular`.

Reply:
1. Let's not hide that kind of partial className in the code. That would need a comment at least but I would avoid doing that altogether: you can check the two classes with 2 `contains` and make the code robust to further refactoring (and even speaking of efficiency, I would not be surprised that two `classList.contains` is faster than a `className.includes` (but not important anyway even if 100x slower)) :wink: 

2. Is not `o_wblog_post_page_cover_regular` always an element that also has the class `o_wblog_post_page_cover`? So only one class check is actually needed? :grimacing:

Reply:
@qsm-odoo Oh indeed, sorry. I just pushed the `classList.contains`.